pom.xml

```diff
@@ -662,7 +661,7 @@
             <dependency>
                 <groupId>ch.qos.logback</groupId>
                 <artifactId>logback-classic</artifactId>
-                <version>1.1.2</version>
+                <version>1.1.3</version>
```

Comment:
Why do we need `logback-classic` be upgraded specifically to `1.1.3`?

Reply:
`cassandra-all` has this dependency with version `1.1.3`. So, the alternative would be to exclude the dependency from `cassandra-all`, but I figured that it makes more sense to just update the patch version here from `1.1.2` to `1.1.3` to avoid this exclusion.